.github/workflows/ci.yaml

```
@@ -64,7 +64,7 @@ jobs:
         awk "/\.\.\./ && ! /Ok$/{exit 1}" logo-unit-test-results.txt
     - name: Archive Test Results
       if: always()
-      uses: actions/upload-artifact@v3
+      uses: actions/upload-artifact@v4
       with:
         name: Logo Unit Test Results
```

I think the following change will fix the current build problem:
```suggestion
        name: Logo Unit Test Results with ${{ matrix.wx_version }}
```

Looks like there was a breaking change in v3 -> v4 around having unique names for artifacts.

---

Thanks for the fix.